setup.py

```diff
@@ -50,7 +50,7 @@
 setup(
     name="mbtest",
     zip_safe=False,
-    version="2.8.1",
+    version="2.8.2",
```

Comment:
This would be a minor release, I think. But no need to make this change - I usually bump this as I actually make the release.

Reply:
Sorry - force of habbit